### PR TITLE
Added api docs [sc-16596]

### PR DESCRIPTION
**Pull request by NehaNaithani**

### CODE REVIEW

File: docs/swagger.json
Comment (baghbidi) [inline docs/swagger.json:1]: Is this file needed?

**Review by baghbidi — APPROVED** [review]

File: docs/swagger.json
Comment (NehaNaithani) [reply docs/swagger.json:1]: yes it is needed, `swagger-ui-express` uses swagger.json to serve the UI.
